compiler-rt/lib/builtins/aarch64/sme-abi.S

```diff
@@ -180,6 +184,48 @@ DEFINE_COMPILERRT_OUTLINE_FUNCTION_UNMANGLED(__arm_za_disable)
   ret
 END_COMPILERRT_OUTLINE_FUNCTION(__arm_za_disable)
 
+DEFINE_COMPILERRT_OUTLINE_FUNCTION_UNMANGLED(__arm_get_current_vg)
+  .variant_pcs __arm_get_current_vg
+  BTI_C
+  mov x0, xzr
```

Comment:
Is this necessary? I'd expect it to be overwritten by the call to `__arm_sme_state`.

Reply:
It does get overwritten, I've removed this